src/components/pages/AboutPage.js

```diff
@@ -39,6 +81,85 @@ function AboutPage({ classes }) {
         <Typography variant="h3" color="inherit">Simple. Fast. Visual.</Typography>
         <Typography variant="h5" className={classes.heroSubtitle}>Course planning for Northwestern University.</Typography>
       </div>
+
+      <div className={classes.section}>
+        <Typography variant="h3" color="primary" className={classes.header}>Serif.nu redefines your scheduling experience.</Typography>
+        <div className={classes.multipleColumns}>
+          <div>
+            <Typography variant="h5" className={classes.blurbs}>No more drawing out your schedules on paper.</Typography>
+            <Typography variant="h5" className={classes.blurbs}>No more clunkly Excel spreadsheets.</Typography>
+            <Typography variant="h5" className={classes.blurbs}>Plan out your next quarter at Northwestern with ease and speed.</Typography>
+          </div>
+          <img src={studentImage} alt="Student with a laptop" className={classes.studentImage} />
+        </div>
+      </div>
+
+      <div className={classes.sectionWithColor}>
+        <Typography variant="h3" color="primary" className={classes.header}>
+          Features
+        </Typography>
+        <div className={classes.multipleColumns}>
+          {/* <img src={screenshot1} alt="Serif screenshot" className={classes.screenshot} /> */}
+          <div>
+            <Typography variant="h5">Search for classes by name </Typography>
+            <Typography variant="h5">Browse by school and subject </Typography>
+            <Typography variant="h5">More coming soon!</Typography>
+          </div>
+        </div>
+      </div>
+
+      <div className={classes.section}>
+        <Typography variant="h3" color="primary" className={classes.header}>The Team</Typography>
+        <Typography variant="h5" className={classes.blurbs}>Serif.nu is run by a team of previous and current Northwestern students dedicated to providing the best scheduling experience.</Typography>
+        <div className={classes.multipleColumns}>
+          <Paper className={classes.studentCard}>
+            <img src={joonProfile} alt="Joon Park" className={classes.profileImage} />
+            <Typography variant="h6">Joon Park</Typography>
+            <Typography variant="subtitle1">Creator & Team Lead</Typography>
+            <Typography variant="body2" className={classes.blurbs}>Software engineer at Grubhub, film composer in another life. Big fan of taking pictures that make me look like I&apos;m falling off of things.</Typography>
+          </Paper>
```

Comment:
Linebreak after each paper for readability?

Reply:
Done